task0/Python_Intro_Ferhat.ipynb

```diff
@@ -665,16 +665,35 @@
   },
```

Comment:
<p>Make it fun for yourselves:</p><p> </p><pre class="ql-syntax" spellcheck="false">age = int(input())
if age &lt; 18:
&nbsp;&nbsp;print("We don't want you here.")
else:
&nbsp;&nbsp;print("You can vote and drink.")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bogaziciai/assignment/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>